data/manifest_core.json

```diff
@@ -666,6 +666,15 @@
           "default": "a"
         }
       },
+      "seasonal-cov": {
+        "resolution": {
```

Comment:
_nit-pick_

I don't think the `resolution` key gets surfaced anywhere on the website (yet?) but I think more appropriate to label these as `virus` or `type`?

Reply:
> I don't think the resolution key gets surfaced anywhere on the website (yet?)

As far as I remember it's only used during when working out what options to expose as potential second trees (part of the `charon/getAvailable` API):

https://github.com/nextstrain/nextstrain.org/blob/7a8e92ffc6448ee1be96186dbdef279c7961d088/src/endpoints/charon/parseManifest.js#L78-L80

(Oh how that needs to be improved one day, but not here!)

Reply:
Is there a list of what are acceptable keys at that level of the data structure, and a key as to what those things mean? Reviewing the whole file, I see @joverlee521 `s point about `"resolution"` not generally being used to distinguish between strains, but also: I don't see `"virus"` used at all, and the only place `"type"` is used is for RSV, where it is common to talk about viral subtypes (at least, that's what a quick web search indicates).

`"virus"` (or `"species"`, I guess?) **do** feel more correct. Feedback welcome.

Reply:
Yeah, the actual value is ignored and just indicates a separate level for the Auspice dropdown. I was mostly flagging for internal reading of the manifest file. 

If we do decide to surface these values in the UI in the future, we'll probably have to do a whole overhaul of the manifest anyways...

Reply:
I think the technically correct word here is "strain". Let's not let this decision hold up this PR!

> the actual value here is mostly ignored

Correct - ignored unless it's "segment".

> If we do decide to surface these values in the UI in the future

I think that was on the cards when Nextstrain was only core datasets, but now that we have such a diverse range of external datasets I don't think we'll build UIs which use manually curated information like this.

Reply:
> I think the technically correct word here is "strain". 

Oh boy. I'm always confused by "strain" vs our use of strain names in Nextstrain 🙈 

> Let's not let this decision hold up this PR!

Yeah, I didn't mean to hold up this PR. Fine to merge as-is.

Reply:
> > I think the technically correct word here is "strain".
> 
> Oh boy. I'm always confused by "strain" vs our use of strain names in Nextstrain 🙈

I don't think "strain" is appropriate here because these are all distinct viral _species_, unless we're using "strain" differently than I think is typical in biology? 

> > Let's not let this decision hold up this PR!
> 
> Yeah, I didn't mean to hold up this PR. Fine to merge as-is.

Merging, still curious about what "strain" means in this context :wink:

Reply:
> I'm always confused by "strain"

To be fair, the entire field is too.

I think any of "subtype" "species" "strain" "cov-strain" is better than "resolution", but it's a minor thing.